src/components/markdown-renderer/markdown-it-plugins/parser-debugger.ts

```diff
@@ -10,7 +10,7 @@ export const MarkdownItParserDebugger: MarkdownIt.PluginSimple = (md: MarkdownIt
   if (process.env.NODE_ENV !== 'production') {
     md.core.ruler.push('test', (state) => {
       console.log(state)
-      return true
+      return false
```

Comment:
Why is this necessary?

Reply:
Not for this PR, but I didn't want to create a new one for this.
(Technically this is more correct, because it tells markdown it, that this plugin didn't process/change the content)